test/Microsoft.ML.Tests/RangeFilterTests.cs

```diff
@@ -29,7 +29,7 @@ public void RangeFilterTest()
             Assert.Equal(2L, cnt);
 
             data = ML.Transforms.Conversion.Hash("Key", "Strings", numberOfBits: 20).Fit(data).Transform(data);
-            var data2 = ML.Data.FilterRowsByKeyColumnFraction(data, "Key", upperBound: 0.5);
+            var data2 = ML.Data.FilterRowsByKeyColumnFraction(data, "Key", upperBound: 0.15);
```

Comment:
>15 [](start = 89, length = 2)

Why is this changing?

Reply:
It's changing because the hash values are different, so the fractional values will also change.

Reply:
I see. Another option would be to keep the `0.5` fraction, and change the number in the `Assert` to match the new value of `cnt`. I'm fine with either :).

---
In reply to: [413200283](https://github.com/dotnet/machinelearning/pull/5013#discussion_r413200283) [](ancestors = 413200283)